include/IMU/MPU6050.h

```diff
@@ -63,6 +63,8 @@ class MPU6050 : public Button::ButtonCallbackInterface
     void worker();
 
     void start();
+    
```

Comment:
Why is this moved to public? Typically, register access shouldn't be accessible to the user of this class.